docs/codeql/codeql-cli/analyzing-databases-with-the-codeql-cli.rst

```diff
@@ -94,36 +94,52 @@ Running a single query
 To run a single query over a CodeQL database for a JavaScript codebase,
 you could use the following command from the directory containing your database::
 
-   codeql database analyze <javascript-database> ../ql/javascript/ql/src/Declarations/UnusedVariable.ql --format=csv --output=js-analysis/js-results.csv 
+   codeql database analyze <javascript-database> ../ql/javascript/ql/src/Declarations/UnusedVariable.ql --format=csv --output=js-analysis/js-results.csv
 
 This command runs a simple query that finds potential bugs related to unused
 variables, imports, functions, or classes---it is one of the JavaScript
 queries included in the CodeQL repository. You could run more than one query by
 specifying a space-separated list of similar paths.
 
 The analysis generates a CSV file (``js-results.csv``) in a new directory
-(``js-analysis``).  
+(``js-analysis``).
 
 You can also run your own custom queries with the ``database analyze`` command.
 For more information about preparing your queries to use with the CodeQL CLI,
 see ":doc:`Using custom queries with the CodeQL CLI <using-custom-queries-with-the-codeql-cli>`."
 
-Running GitHub code scanning suites
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Running a CodeQL pack
+~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: ../reusables/beta-note-package-management.rst
+
+To run an existing CodeQL query pack from the GitHub Container registry, you need to download it first::
+
+   codeql pack download microsoft/coding-standards@1.0.0
+
+Afterwards, you can run the pack on a specific database::
+
+   codeql database analyze <database> microsoft/coding-standards@1.0.0 <scope>/<other-pack> --format=sarifv2.1.0 --output=query-results.sarif
```

Comment:
@aeisenberg, I am curious about this example. I went to https://github.com/orgs/microsoft/packages?tab=packages&q= and can not see this pack there. Are they going to publish it later?

Reply:
May I suggest to use something like `codeql database analyze <database> codeql/csharp-queries <scope>/<other-pack> --format=sarifv2.1.0 --output=query-results.sarif` example instead?